node_modules/.package-lock.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
This entire node_modules directory should *not* be versioned. Remove this directory from git.